src/main/java/com/vmware/vipclient/i18n/messages/service/ProductService.java

```diff
@@ -77,7 +73,7 @@ public ProductService(BaseDTO dto) {
      *
      * @return list of locales of the product specified in the dto object
      */
-    public List<String> getSupportedLocales(){
+    public List<String> getSupportedLanguageTags(){
```

Comment:
Suggest not to change this method but to name new method to another name.
```suggestion
    public List<String> getSupportedlocales(){
```

Reply:
See my comment above.